subprojects/core/src/integTest/groovy/org/gradle/api/tasks/TaskInputPropertiesIntegrationTest.groovy

```diff
@@ -19,11 +19,12 @@ package org.gradle.api.tasks
 import org.gradle.api.file.FileCollection
 import org.gradle.api.provider.Provider
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.integtests.fixtures.DirectoryBuildCacheFixture
 import org.gradle.internal.Actions
 import spock.lang.Issue
 import spock.lang.Unroll
 
-class TaskInputPropertiesIntegrationTest extends AbstractIntegrationSpec {
+class TaskInputPropertiesIntegrationTest extends AbstractIntegrationSpec implements DirectoryBuildCacheFixture {
```

Comment:
Instead of implementing DirectoryBuildCacheFixture that enabled caching for every test case, you could use a `TestBuildCache` instance.

Reply:
I am using `TestBuildCache` now. I still use `BuildCacheFixture` to get the nice `withBuildCache()` method.